modules/cnf-gathering-data-about-cluster-using-must-gather.adoc

```diff
@@ -21,6 +21,13 @@ In earlier versions of {product-title}, the Performance Addon Operator provided
 
 .Procedure
 
+. Check if a label exists for a Machine Config Pool (MCP), that matches with the MCP name:
```

Comment:
* I think this can be lowercased, right? "machine config pool (MCP)"
* Best to check with the SME, but I think the below command is actually labeling, not just checking if a label exists. If that's so, then you'd likely need to reword this step a bit.
* Don't think you need the comma. But it's also a bit unclear (at least to me) with what MCP name you're talking about here

Reply:
@jeniferh Can you please confirm if the command is adding a label as well and not just checking if a label exists? This question is related to point 2.
Also, wrt point 3 - can you confirm if we are looking for any specific MCP name?

Reply:
Thank you very much! @jeniferh 
As per the explanation, I updated the PR with an extra optional step to verify if the label exists or not. Also added the (required) next step to add the MCP label. Please let me know if that looks good.

Reply:
Thanks! I think one little change in Step2 may make it more clear because of the optional Step1:
"Add a label " change to: "If a matching label does not exist, add a label[...]"

Reply:
Addressed. Thanks